frontend/src/common/authentication/auth-walls/IDIRAuthWall.tsx

```diff
@@ -84,7 +84,7 @@ export const IDIRAuthWall = ({
     }
 
     const doesUserHaveAccess = DoesUserHaveAuthGroup<IDIRUserAuthGroupType>({
```

Comment:
Maybe rename "DoesUserHaveAuthGroup" to "DoesUserHaveRole" instead? And as mentioned earlier, maybe rename "IDIRUserAuthGroupType" to "IDIRUserRole"?